python/GafferSceneTest/AttributeVisualiserTest.py

```diff
@@ -163,8 +162,8 @@ def testShaderNodeColorMode( self ) :
 
 	def testVectors( self ) :
 
-		fileName = os.path.join( self.temporaryDirectory(), "attributes.scc" )
-		scene = IECoreScene.SceneInterface.create( fileName, IECore.IndexedIO.Write )
+		filePath = self.temporaryDirectory() / "attributes.scc"
+		scene = IECoreScene.SceneInterface.create( filePath.as_posix(), IECore.IndexedIO.Write )
```

Comment:
Is the `as_posix()` here actually needed, instead of `str( filePath )`? Looking through the Cortex source I would have guessed it would have worked with native format too. I think we want to limit `as_posix()` to cases where the use of the generic format (`/`) is a hard requirement.

Reply:
Ah yes, paths going into Cortex can mostly use `str( filePath )` for os-native formats. I've fixed those in 75510d5a3a3ec430a94c00553f283b6a26cb924b, with the exception of `OpenGLRenderTest` as noted below.

Reply:
Thanks! I want to avoid the situation where we have `as_posix()` scattered around in a cargo-culted sort of way, so that any remaining ones stick out like sore thumbs for us to fix, and/or for folks to learn from.